Lib/test/test_minidom.py

```diff
@@ -1152,6 +1152,19 @@ def testEncodings(self):
 
         doc.unlink()
 
+    def testStandalone(self):
+        doc = parseString('<foo>&#x20ac;</foo>')
+        self.assertEqual(doc.toxml(),
+                         '<?xml version="1.0" ?><foo>\u20ac</foo>')
+        self.assertEqual(doc.toxml(standalone=None),
+                         '<?xml version="1.0" ?><foo>\u20ac</foo>')
+        self.assertEqual(doc.toxml(standalone=True),
+             '<?xml version="1.0" standalone=\'yes\'?><foo>\u20ac</foo>')
+        self.assertEqual(doc.toxml(standalone=False),
+             '<?xml version="1.0" standalone=\'no\'?><foo>\u20ac</foo>')
```

Comment:
I think both values should use the same quotes here. Since both the version and the encoding used double quotes, please also use double quotes for `standalone`.

Please also add a test that declares an encoding.

Reply:
Yup, missed the test case with an encoding. 
Added and the quote styles are made consistent either.